rcqt.so/src/action.cpp

```diff
@@ -106,6 +107,7 @@ rcqt_action::~rcqt_action() {
 int rcqt_action::run() {
   string msg;
   bool pkgchk_bool = false;
+  bool pkglist_bool = false;
```

Comment:
Can we do away with having  data type in variable name? i.e., "bool" in name

Reply:
Yes